### PR TITLE
Fix tutorial api

### PR DESCRIPTION
**Pull request by toofff**

### CODE REVIEW

File: tutorial/api.md [hunk 1]
```diff
@@ -89,10 +91,16 @@ Run the built-in web server:
 You can open `http://localhost:8000` with you preferred REST client.
 We recommend [Postman](https://www.getpostman.com), and you will see later that API Platform is nicely integrated with it.
 
+### With docker
```
Comment (dunglas) [inline tutorial/api.md]: Docker instead of docker.

File: tutorial/api.md [hunk 2]
```diff
@@ -89,10 +91,16 @@ Run the built-in web server:
...
+
+You can also use [docker](https://www.docker.com/) for the installation of your project.
```
Comment (dunglas) [inline tutorial/api.md]: Same here. And I would write "to install your project".

**Comment by dunglas** [comment]
I left some minor comments to fix. Thanks for those improvements!

File: tutorial/api.md (hunk 1 above)
Comment (theofidry) [inline tutorial/api.md]: the `Composer` and not `composer` for the previous chapter :)

File: tutorial/api.md [hunk 3]
```diff
@@ -89,10 +91,16 @@ Run the built-in web server:
...
+
+it just make a clone of the repository Api Platform and [follow documentation docker](../deployment/docker.md)
+
 When you're done with the demo app and want to create your own API:
```
Comment (theofidry) [inline tutorial/api.md]: I would add another section: this part is not specific to docker but the way it is right now makes you think so which is a bit confusing

**Pull request closed without merging**